en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -9,6 +9,8 @@ icon: table
 [GridField](api:SilverStripe\Forms\GridField\GridField) is Silverstripe CMS's implementation of data grids. The main purpose of this field type is to display
 tabular data in a format that is easy to view and modify. It can be thought of as a HTML table with some tricks.
 
+Usually `GridField` is used with `DataObject` records - but it can be used with data that isn't represented by `DataObject` records as well. See [using `GridField` with arbitrary data](/developer_guides/forms/using_gridfield_with_arbitrary_data/) for more information.
```

Comment:
Left align links.

```suggestion
Usually `GridField` is used with `DataObject` records - but it can be used with data that isn't represented by `DataObject` records as well. 

See [using `GridField` with arbitrary data](/developer_guides/forms/using_gridfield_with_arbitrary_data/) for more information.
```

Reply:
Done